docs/getting-started/overview.rst

```diff
@@ -1,6 +1,45 @@
-.. include:: /stub-topic.txt
+Overview of different editions of .NET
+======================================
+
+With the advent of .NET Core and .NET Native, the layout of the .NET ecosystem has changed significantly. Where before there was a (somewhat) simple mapping between platforms and .NET Framework versions, today we have a lot of 
+
+.NET Framework
+--------------
+The tried and true workhorse of the past 10+ years, the full .NET framework is still around, of course, and has been recently updated to version 4.6. There are many enhancements to the CLR (the runtime) and the BCL (the class library) that comes with this release for the consumers presented in the diagram above, such as the .NET Compiler Platform, new language innovations etc.
+
+This edition of the .NET Framework is not cross-platform and you can run it only on Windows. Most of the consumers of this particular edition of the .NET Framework should be already know, if in name only. Windows Presentation Foundation or WPF is a framework for building desktop applications using specific technologies to make them compelling and rich. Windows Forms is already known as the bulwark of the LOB desktop applications from the early day of the .NET Framework. Of course, ASP.NET is Microsoft's main web stack, which is getting a major overhaul in the 5 release timeframe.
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, pay-as-you-go, cross-platform port of the .NET Framework. The roadmap currently in place covers support for three main operating systems, Linux, Windows and OS X. The community porters have made it work on FreeBSD as well. It is freely available on `GitHub <https://www.github.com/dotnet/coreclr>`_ and it is very easy to get for the supported systems and get going.
```

Comment:
I think we covered "pay as you go" on another PR.

"cross-platform port" makes it sound like it is a high-fidelity port of the .NET Framework, both in terms of completeness and behavior. This isn't true.

Instead of "community porters", I'd just say "the community". I haven't heard the term "porters" before, so we should avoid it. The FreeBSD port isn't done. I'd prefer language like: "We expect the community to port .NET Core to other operating systems. A [FreeBSD](https://github.com/dotnet/coreclr/blob/master/Documentation/building/freebsd-instructions.md) port is in progress".

I think there is too much going on in the last sentence. Also, what "it" refers to is ambiguous.